sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/TableIdentifierParserSuite.scala

```diff
@@ -48,7 +48,7 @@ class TableIdentifierParserSuite extends SparkFunSuite {
     "false", "fetch", "float", "for", "grant", "group", "grouping", "import", "in",
     "insert", "int", "into", "is", "pivot", "lateral", "like", "local", "none", "null",
     "of", "order", "out", "outer", "partition", "percent", "procedure", "range", "reads", "revoke",
-    "rollup", "row", "rows", "set", "smallint", "table", "timestamp", "to", "trigger",
+    "rollup", "row", "rows", "set", "smallint", "table", "query", "timestamp", "to", "trigger",
```

Comment:
super nit: if with alphabet order, put it between "procedure" and "range" may be better :)